services/search/testing/test_search_engine.py

```diff
@@ -19,16 +21,17 @@ def test_double_weights_synonyms():
     """
     result = {1: 4}
 
-    assert result == SearchEngine(synonyms_weight=2).search_query(constants.KEYWORDS_ARTICLE_1,
-                                                                  constants.SYNONYMS_ARTICLE_1)
-
-# I mocked the db endpoint to only be for article 1
-# therefore this test is not yet available without the endpoint
-#
-# def test_multiple_articles():
-#    """
-#    Have the query apply for two articles
-#    """
-#    result = {3: 2, 4: 2}
-#    assert result == SearchEngine(constants.ARTICLES).search_query(constants.KEYWORDS_MULTIPLE,
-#                                                                   constants.SYNONYMS_MULTIPLE)
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MOCK_1):
+        assert result == SearchEngine(synonyms_weight=2).search_query(constants.KEYWORDS_ARTICLE_1,
+                                                                      constants.SYNONYMS_ARTICLE_1)
+
+
+def test_multiple_articles():
+    """
+    Have the query apply for two articles
+    """
+    result = {3: 2, 4: 2}
+
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MULTIPLE):
+        assert result == SearchEngine().search_query(constants.KEYWORDS_MULTIPLE,
+                                                     constants.SYNONYMS_MULTIPLE)
```

Comment:
Good test, as long as this data is combined into a single array, it should be fine on my end.

Reply:
This is the search endpoint, only oscar should be calling it